doc/stdlib/base.rst

```diff
@@ -977,53 +977,43 @@ System
 
    Reference to the singleton ``EnvHash``\ , providing a dictionary interface to system environment variables.
 
-.. function:: @unix
+.. function:: is_unix([os])
 
    .. Docstring generated from Julia source
 
-   Given ``@unix? a : b``\ , do ``a`` on Unix systems (including Linux and OS X) and ``b`` elsewhere. See documentation in :ref:`Handling Operating System Variation <man-handling-operating-system-variation>`\ .
+   Predicate for testing if the OS provides a Unix-like interface. See documentation :ref:``Handling Operating System Variation <man-handling-operating-system-variation>`` .
 
-.. function:: @unix_only
+.. function:: is_apple([os])
 
    .. Docstring generated from Julia source
 
-   A macro that evaluates the given expression only on Unix systems (including Linux and OS X). See documentation in :ref:`Handling Operating System Variation <man-handling-operating-system-variation>`\ .
+   Predicate for testing if the OS is a derivative of Apple Macintosh OS X or Darwin. See documentation :ref:``Handling Operating System Variation <man-handling-operating-system-variation>`` .
 
-.. function:: @osx
+.. function:: is_linux([os])
 
    .. Docstring generated from Julia source
 
-   Given ``@osx? a : b``\ , do ``a`` on OS X and ``b`` elsewhere. See documentation in :ref:`Handling Operating System Variation <man-handling-operating-system-variation>`\ .
+   Predicate for testing if the OS is a derivative of Linux. See documentation :ref:``Handling Operating System Variation <man-handling-operating-system-variation>`` .
 
-.. function:: @osx_only
+.. function:: is_bsd([os])
 
    .. Docstring generated from Julia source
 
-   A macro that evaluates the given expression only on OS X systems. See documentation in :ref:`Handling Operating System Variation <man-handling-operating-system-variation>`\ .
+   Predicate for testing if the OS is a derivative of BSD. See documentation :ref:``Handling Operating System Variation <man-handling-operating-system-variation>`` .
 
-.. function:: @linux
+.. function:: windows_version()
```

Comment:
isn't the genstdlib.jl supposed to warn about that? I hadn't checked since I assumed that's what all the "missing doc for" messages were intended to catch.